### DISCUSSION TITLE
Payment issue in parallel reduce

### DISCUSSION THREAD

**Discussion by Jace254** [reach-sh/reach-lang#1295] [General]
![Capture](https://user-images.githubusercontent.com/47209391/181099929-ade85bf8-cf66-4cd9-bd91-225d8743dc62.PNG)

I get this error when I try paying in portions within a parallel reduce. 
Here is the Repo :
[https://github.com/Jace254/MapsandSets/tree/5515e72b7c11a1790b96a985a1d042eed48bfcab](url)

Is this supposed to happen?

Edit: I added a check to ascertain that the balance is greater than or equal to the payment amount at compilation time but this time it resulted to an error with the compiler

current commit here:

[https://github.com/Jace254/MapsandSets/tree/payment-issue](url)
![Payment  Issue](https://user-images.githubusercontent.com/47209391/181196289-8be8fb13-d450-4da6-8b56-3087b6a4a430.PNG)

**Comment by jeapostrophe** [comment]
I answered in the Issue you created.
